main.py

```diff
@@ -12,6 +12,42 @@
 
 # Import whatever libraries/modules you need
```

Comment:
Try to break up your code into different functions, in this case, it can be data_preparation(), model_construction(),model_training() and etc. You can be creative about the names. The purpose is that when you are actually contributing to our repo, you can write some clear and easy-to-debug code.